aten/src/TH/generic/THTensorEvenMoreMath.cpp

```diff
@@ -78,6 +78,76 @@ accreal THTensor_(sumall)(THTensor *tensor)
     scalar_t, tensor, *tensor_data, sum, UNCERTAIN_TH_OMP_OVERHEAD_THRESHOLD);
   return sum;
 }
+
```

Comment:
Was the code in this block changed?

Reply:
Here and everywhere else in this PR, i just moved methods up above the `#if !defined(TH_REAL_IS_BOOL)` to enable them for bool.